benchmark/buffers/buffer-tojson.js

```diff
@@ -4,7 +4,7 @@ const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
   n: [1e4],
-  len: [0, 10, 256, 4 * 1024]
+  len: [0, 256, 4 * 1024]
```

Comment:
As above: super tiny buffers should not have any influence. Likely all of these length are not testing cases that would ever make a difference in a real application.